spec/cerner/oauth1a/protocol_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+require 'cerner/oauth1a/protocol'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe Cerner::OAuth1a::Protocol do
+  describe '.parse_url_query_string' do
```

Comment:
Some specs are using `'#method_name'` and others are doing `'.method_name'`. They should probably be consistent, but I'm not sure which way is preferred.

Reply:
'#' is used for when referring to an instance method. '.' is used for when referring to a class/module method. This spec is testing a module method.